### ISSUE TITLE
Cannot use FFB Wheel in Windows10 

### ISSUE THREAD

**Issue by Xx-yuta-xX** [necroware/gameport-adapter#78]
I created a gameport-adapter and tried to use FFB Wheel on Windows 10 but could not use it.
![Screenshot 2023-12-27 191225](https://github.com/necroware/gameport-adapter/assets/153920858/f9ed24d3-636f-45c1-97ec-181a916192b4)

![Screenshot 2023-12-27 191322](https://github.com/necroware/gameport-adapter/assets/153920858/13b09a10-9974-4e08-b36c-5dc7ea20146d)

**Comment by creopard** [comment]
Please try the firmware files from the current repository instead of the release 2.1.1 (from 2022).
The current repo files contain fixes regarding Sidewinder timings.

**Comment by Xx-yuta-xX** [comment]
I tried but could not work.

**Comment by necroware** [comment]
This is probably a duplicate of the issue #70 

Please try following branch, we still don't know if that one works.
https://github.com/necroware/gameport-adapter/tree/win10-usb-hid-fix

**Comment by Xx-yuta-xX** [comment]
I could fix the problem thank you.

**Comment by necroware** [comment]
Open source projects work only if everybody helps. Did the code from that branch help, or did you do something else to get it working? If that branch helped, I will integrate it into the main branch, but I can do it only after confirmation from s.o. who has this issue. It seems, that not everybody who has windows 10 runs into this problem, that's why I need a feedback from s.o. who is affected.

**Comment by Xx-yuta-xX** [comment]
The branch code had a bug. Then I rewrite "allign" to align. That is all I did.

**Comment by necroware** [comment]
Ok, then I fix that and merge into the main.
